src/main.py

```diff
@@ -118,6 +163,16 @@ def handleRequest(req) -> Response:
 
 
 def handleConnection(conn, addr):
+    '''Receives the HTTP request and send an appropriated response
+
+    Receive an message of limited size, tries to perform the action desired by the
+    client and sends the response.
+
+    If any error was found during the process, send an notifying response to the client
```

Comment:
If any error _is_ found during the process, _sends_ a notifying response to the client